src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -255,44 +256,46 @@ class HoldRequest extends React.Component {
     }
 
     return (
-      <div id="mainContent">
-        <div className="nypl-request-page-header">
-          <div className="nypl-full-width-wrapper">
-            <div className="row">
-              <div className="nypl-column-full">
-                <Breadcrumbs
-                  query={`q=${searchKeywords}`}
-                  bibUrl={`/bib/${bibId}`}
-                  type="hold"
-                />
-                <h2>{appConfig.displayTitle}</h2>
+      <DocumentTitle title={`${title} | Shared Collection Catalog | NYPL`}>
```

Comment:
According to #675 , this should be `Item Request | Shared Collection Catalog | NYPL`.